packages/x-archetype-utils/package.json

```diff
@@ -33,13 +33,23 @@
     "test": "jest",
     "prepublishOnly": "npm run build"
   },
+  "dependencies": {
+    "@empathyco/x-deep-merge": "^1.3.0-alpha.19",
+    "@empathyco/x-utils": "^1.0.0-alpha.5"
```

Comment:
This dependency is only use for testing purposes. I think it can be moved to the `devDependencies` section.

Maybe I'm wrong and setting it as a `peerDependency` is enough, but... shouldn't `VueI18n` be moved to the `dependencies` section? It is `imported` in the `i18n.plugin.ts` and we're creating a VueI18n instance there.